packages/terra-action-footer/src/terra-dev-site/doc/example/centered/Borderless.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Button from 'terra-button';
+import Spacer from 'terra-spacer';
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
+import CenteredActionFooter from '../../../../../src/CenteredActionFooter';
```

Comment:
This needs to be `import ActionFooter from 'terra-action-footer/lib/ActionFooter`. Our hot reloading can resolve this during testing, however we need this import to be correctly resolved for the terra-ui site.